images/ingress-nginx-controller/configs/latest.apko.yaml

```diff
@@ -0,0 +1,196 @@
+      "org.opencontainers.image.source" : "https://github.com/chainguard-images/images/tree/main/images/ingress-nginx-controller"
+
+environment:
+  PATH: "$PATH:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/luajit/bin:/usr/local/nginx/sbin/"
+  LUA_PATH: "/usr/local/share/luajit-2.1.0-beta3/?.lua;/usr/local/share/lua/5.1/?.lua;/usr/local/lib/lua/?.lua;/usr/lib/lua/?.lua;"
+  LUA_CPATH: "/usr/local/lib/lua/5.1/?.so;/usr/local/lib/lua/?/?.so;/usr/local/lib/lua/?.so;/usr/lib/lua/?.lua;"
+  LD_PRELOAD: "/usr/lib/libmimalloc.so"
+
+work-dir:  /etc/nginx
+
+entrypoint:
+  cmd: /nginx-ingress-controller
+cmd: "/usr/bin/dumb-init ---"
+
+paths:
+  - path: /etc/ingress-controller
+    type: directory
+    permissions: 0o755
+    uid: 101
+    gid: 101
+  - path: /etc/ingress-controller/ssl
+    type: directory
+    permissions: 0o755
+    uid: 101
+    gid: 101
+    recursive: true
+  - path: /etc/ingress-controller/auth
+    type: directory
+    permissions: 0o755
+    uid: 101
+    gid: 101
+    recursive: true
+  - path: /usr/local/
+    type: directory
+    permissions: 0o755
+    uid: 101
+    gid: 101
+    recursive: true
+  - path: /usr/local/
+    type: directory
+    permissions: 0o755
+    uid: 101
+    gid: 101
+    recursive: true
+  - path: /var/lib/
+    type: directory
+    permissions: 0o755
+    uid: 101
+    gid: 101
+    recursive: true
+  - path: /var/lib/nginx/
+    type: directory
+    uid: 101
+    gid: 101
+    permissions: 0o755
+    recursive: true
+  - path: /var/lib/nginx/body
+    type: directory
+    uid: 101
+    gid: 101
+    permissions: 0o755
+    recursive: true
+  - path: /usr/local/nginx
+    type: directory
+    uid: 101
+    gid: 101
+    permissions: 0o755
+    recursive: true
+  - path: /run/nginx
+    uid: 101
+    gid: 101
+    type: directory
+    permissions: 0o755
+    recursive: true
+  - path: /etc/nginx/modules
+    uid: 101
+    gid: 101
+    type: directory
+    permissions: 0o755
+    recursive: true
+  - path: /etc/nginx
+    uid: 101
+    gid: 101
+    type: directory
+    permissions: 0o755
+    recursive: true
+  - path: /usr/local/nginx
+    uid: 101
+    gid: 101
+    type: directory
+    permissions: 0o755
+    recursive: true
+  - path: /opt/modsecurity/
+    uid: 101
+    gid: 101
+    type: directory
+    permissions: 0o755
+    recursive: true
+  - path: /var/log
+    uid: 101
+    gid: 101
+    type: directory
+    permissions: 0o755
+  - path: /tmp/nginx
+    uid: 101
+    gid: 101
+    type: directory
+    permissions: 0o755
+    recursive: true
+  - path: /sbin/nginx
+    source: /usr/local/nginx/sbin/nginx
+    type: symlink
+    uid: 101
+    gid: 101
+    permissions: 0o755
+  - path: /usr/lib/libmimalloc.so
+    source: /usr/lib/libmimalloc.so.1.7
+    type: symlink
+    uid: 101
+    gid: 101
+    permissions: 0o755    
+  - path: /usr/local/lib/libmimalloc.so
+    source: /usr/lib/libmimalloc.so.1.7
```

Comment:
symlinks tend to go directly into the apk package, that way we don't have to maintain the soname version here when the mimalloc package update brings in a new version.